packages/babel-plugin-transform-flow-comments/test/fixtures/flow-comments/class-type-parameter-extends-type-parameter/input.js

```diff
@@ -0,0 +1 @@
+class Foo<T> extends Bar<R> {}
```

Comment:
Could you add a test when there is a comment between class identifier and extends keyword?
```js
class Foo<T> /* inner */ extends Bar<R> {}
```

Reply:
Thanks you for this case, unfortunately it was transformed into:
```js
class Foo
/*:: <T>*/
extends
/* inner */
Bar
/*:: <R>*/
{}
```
Which should have been
```js
class Foo
/*:: <T>*/
/* inner */
extends Bar
/*:: <R>*/
{}
```
Looking into it... :confused:

Reply:
Fixed in 422685f